app/components/mig/login/login.styl

```diff
@@ -1,3 +1,50 @@
+$mantis = #3cb9a0
+$carrotOrange = #ec932b
```

Comment:
You are using this colour in **buttons.styl**. Might want to put the colour variables into their own file and import them into both.